google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/SqlTestScriptsGenerator.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spanner.jdbc;
+
+/** Class that runs all generators of SQL test scripts for the Connection API */
+public class SqlTestScriptsGenerator {
+
+  /** Main method for generating the test script */
+  public static void main(String[] args) throws Exception {
```

Comment:
can you please explain how this is used? Do we need a main method here?

Reply:
This is only used when we need to generate new SQL test scripts. This is necessary if we change anything in the client side SQL dialect, such as adding new statements or changing the behavior of existing statements. The generated SQL test scripts could also be used for client libraries of other programming languages if were to implement a similar connection based API for one of them.
The main method can be invoked by running the following command:

```
mvn -P generate-test-sql-scripts compile
```

This is also documented in the `ConnectionImplGeneratedSqlScriptTest` class.